index.html

```diff
@@ -1,19 +1,71 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <head>
+    <meta charset="UTF-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
     <title>Karma</title>
-    <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
-    <link rel="stylesheet" href="css/normalize.css">
-    <link rel="stylesheet" href="css/style.css">
-    <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
-</head>
-<body>
-
+    <link
```

Comment:
Hi @MonMon20 
Well done for using:

1)semantic HTML
2)HTML nesting and indentation 
3)your page looks great 🥳 🎊

what you need to improve: 👩🏼‍💻

1) you did not use classes and you just used the tag names in your CSS file, please change all of them and use classes instead.
2)you did not use flex-box, I just saw one display: flex in ul tag and one of the objectives of this homework was for you to practice more about classes and flex-box.
3) you used width and height for images, instead of doing this you have to use a div (wrapper) and wrap your images with that div and give each of the img tags a width depending on the width of the div tag.
4) you did not use classes for your parent elements.
5) in your HTML file, for some of the elements you wrote some classes but in your CSS file you do not have any class with that name(you did not create any classes) which will make confusion, it is better whenever you want to create a class, just choose  a meaningful name for your class and create it in your CSS file at the same time ( to prevent to forget it somehow 😉 )